--- openssl-3.0.5/Configurations/unix-Makefile.tmpl.orig	2022-07-10 17:18:36.338376691 +0200
+++ openssl-3.0.5/Configurations/unix-Makefile.tmpl	2022-07-10 17:29:34.034813643 +0200
@@ -304,8 +304,8 @@ LIBDIR={- our $libdir = $config{libdir};
 # $(libdir) is chosen to be compatible with the GNU coding standards
 libdir={- file_name_is_absolute($libdir)
           ? $libdir : '$(INSTALLTOP)/$(LIBDIR)' -}
-ENGINESDIR=$(libdir)/engines-{- $sover_dirname -}
-MODULESDIR=$(libdir)/ossl-modules
+ENGINESDIR=/$(LIBDIR)/engines-{- $sover_dirname -}
+MODULESDIR=/$(LIBDIR)/ossl-modules
 
 # Convenience variable for those who want to set the rpath in shared
 # libraries and applications
