--- openssl-1.1.1/Configurations/unix-Makefile.tmpl~	2018-09-13 13:40:10.000000000 +0200
+++ openssl-1.1.1/Configurations/unix-Makefile.tmpl	2018-09-13 13:40:42.065819616 +0200
@@ -170,7 +170,7 @@ LIBDIR={- our $libdir = $config{libdir};
 # $(libdir) is chosen to be compatible with the GNU coding standards
 libdir={- file_name_is_absolute($libdir)
           ? $libdir : '$(INSTALLTOP)/$(LIBDIR)' -}
-ENGINESDIR=$(libdir)/engines-{- $sover_dirname -}
+ENGINESDIR=/$(LIBDIR)/engines-{- $sover_dirname -}
 
 # Convenience variable for those who want to set the rpath in shared
 # libraries and applications
@@ -888,7 +888,7 @@ libcrypto.pc:
 	        echo 'libdir=$(libdir)'; \
 	    fi; \
 	    echo 'includedir=$${prefix}/include'; \
-	    echo 'enginesdir=$${libdir}/engines-{- $sover_dirname -}'; \
+	    echo 'enginesdir=/$(LIBDIR)/engines-{- $sover_dirname -}'; \
 	    echo ''; \
 	    echo 'Name: OpenSSL-libcrypto'; \
 	    echo 'Description: OpenSSL cryptography library'; \
