--- openssl-0.9.8zh/crypto/cryptlib.h~	2015-12-03 16:55:31.000000000 +0200
+++ openssl-0.9.8zh/crypto/cryptlib.h	2015-12-03 18:00:16.202974639 +0200
@@ -82,7 +82,7 @@
 # ifndef OPENSSL_SYS_VMS
 #  define X509_CERT_AREA          OPENSSLDIR
 #  define X509_CERT_DIR           OPENSSLDIR "/certs"
-#  define X509_CERT_FILE          OPENSSLDIR "/cert.pem"
+#  define X509_CERT_FILE          "/etc/certs/ca-certificates.crt"
 #  define X509_PRIVATE_DIR        OPENSSLDIR "/private"
 # else
 #  define X509_CERT_AREA          "SSLROOT:[000000]"
