--- openssl-1.1.1d/Configurations/unix-Makefile.tmpl.orig	2019-09-27 15:57:40.580222104 +0200
+++ openssl-1.1.1d/Configurations/unix-Makefile.tmpl	2019-09-27 16:03:43.774921176 +0200
@@ -201,77 +201,6 @@
    "" -}
 # Do not edit these manually. Use Configure with --prefix or --openssldir
 # to change this!  Short explanation in the top comment in Configure
-INSTALLTOP_dev={- # $prefix is used in the OPENSSLDIR perl snippet
-                  #
-                  use File::Spec::Win32;
-                  my $prefix_default = "$mingw_installroot/OpenSSL";
-                  our $prefix =
-                      File::Spec::Win32->canonpath($config{prefix}
-                                                  || $prefix_default);
-                  our ($prefix_dev, $prefix_dir, $prefix_file) =
-                      File::Spec::Win32->splitpath($prefix, 1);
-                  $prefix =~ s|\\|/|g;
-                  $prefix_dir =~ s|\\|/|g;
-                  $prefix_dev -}
-INSTALLTOP_dir={- my $x = File::Spec::Win32->canonpath($prefix_dir);
-                  $x =~ s|\\|/|g;
-                  $x -}
-OPENSSLDIR_dev={- #
-                  # The logic here is that if no --openssldir was given,
-                  # OPENSSLDIR will get the value "$mingw_commonroot/SSL".
-                  # If --openssldir was given and the value is an absolute
-                  # path, OPENSSLDIR will get its value without change.
-                  # If the value from --openssldir is a relative path,
-                  # OPENSSLDIR will get $prefix with the --openssldir
-                  # value appended as a subdirectory.
-                  #
-                  use File::Spec::Win32;
-                  our $openssldir =
-                      $config{openssldir} ?
-                          (File::Spec::Win32->file_name_is_absolute($config{openssldir}) ?
-                               File::Spec::Win32->canonpath($config{openssldir})
-                               : File::Spec::Win32->catdir($prefix, $config{openssldir}))
-                          : File::Spec::Win32->canonpath("$mingw_commonroot/SSL");
-                  our ($openssldir_dev, $openssldir_dir, $openssldir_file) =
-                      File::Spec::Win32->splitpath($openssldir, 1);
-                  $openssldir =~ s|\\|/|g;
-                  $openssldir_dir =~ s|\\|/|g;
-                  $openssldir_dev -}
-OPENSSLDIR_dir={- my $x = File::Spec::Win32->canonpath($openssldir_dir);
-                  $x =~ s|\\|/|g;
-                  $x -}
-LIBDIR={- our $libdir = $config{libdir} || "lib";
-          File::Spec::Win32->file_name_is_absolute($libdir) ? "" : $libdir -}
-ENGINESDIR_dev={- use File::Spec::Win32;
-                  our $enginesdir =
-                      File::Spec::Win32->catdir($prefix,$libdir,
-                                                "engines-$sover_dirname");
-                  our ($enginesdir_dev, $enginesdir_dir, $enginesdir_file) =
-                      File::Spec::Win32->splitpath($enginesdir, 1);
-                  $enginesdir =~ s|\\|/|g;
-                  $enginesdir_dir =~ s|\\|/|g;
-                  $enginesdir_dev -}
-ENGINESDIR_dir={- my $x = File::Spec::Win32->canonpath($enginesdir_dir);
-                  $x =~ s|\\|/|g;
-                  $x -}
-# In a Windows environment, $(DESTDIR) is harder to contatenate with other
-# directory variables, because both may contain devices.  What we do here is
-# to adapt INSTALLTOP, OPENSSLDIR and ENGINESDIR depending on if $(DESTDIR)
-# has a value or not, to ensure that concatenation will always work further
-# down.
-ifneq "$(DESTDIR)" ""
-INSTALLTOP=$(INSTALLTOP_dir)
-OPENSSLDIR=$(OPENSSLDIR_dir)
-ENGINESDIR=$(ENGINESDIR_dir)
-else
-INSTALLTOP=$(INSTALLTOP_dev)$(INSTALLTOP_dir)
-OPENSSLDIR=$(OPENSSLDIR_dev)$(OPENSSLDIR_dir)
-ENGINESDIR=$(ENGINESDIR_dev)$(ENGINESDIR_dir)
-endif
-
-# $(libdir) is chosen to be compatible with the GNU coding standards
-libdir={- File::Spec::Win32->file_name_is_absolute($libdir)
-          ? $libdir : '$(INSTALLTOP)/$(LIBDIR)' -}
 {- output_on() if $config{target} !~ /^mingw/; "" -}
 
 MANDIR=$(INSTALLTOP)/share/man
