--- openssl-1.1.1/include/internal/cryptlib.h~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/include/internal/cryptlib.h	2018-09-11 22:14:32.008012409 +0200
@@ -56,7 +56,7 @@ DEFINE_LHASH_OF(MEM);
 # ifndef OPENSSL_SYS_VMS
 #  define X509_CERT_AREA          OPENSSLDIR
 #  define X509_CERT_DIR           OPENSSLDIR "/certs"
-#  define X509_CERT_FILE          OPENSSLDIR "/cert.pem"
+#  define X509_CERT_FILE          "/etc/certs/ca-certificates.crt"
 #  define X509_PRIVATE_DIR        OPENSSLDIR "/private"
 #  define CTLOG_FILE              OPENSSLDIR "/ct_log_list.cnf"
 # else
