diff -urN openssl-1.1.1/doc.org/man1/asn1parse.pod openssl-1.1.1/doc/man1/asn1parse.pod
--- openssl-1.1.1/doc.org/man1/asn1parse.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/asn1parse.pod	2018-09-11 22:09:16.148714293 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-asn1parse,
-asn1parse - ASN.1 parsing tool
+openssl-asn1parse - ASN.1 parsing tool
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/ca.pod openssl-1.1.1/doc/man1/ca.pod
--- openssl-1.1.1/doc.org/man1/ca.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/ca.pod	2018-09-11 22:08:39.447633905 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-ca,
-ca - sample minimal CA application
+openssl-ca - sample minimal CA application
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/ciphers.pod openssl-1.1.1/doc/man1/ciphers.pod
--- openssl-1.1.1/doc.org/man1/ciphers.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/ciphers.pod	2018-09-11 22:09:10.135203937 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-ciphers,
-ciphers - SSL cipher display and cipher list tool
+openssl-ciphers - SSL cipher display and cipher list tool
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/cms.pod openssl-1.1.1/doc/man1/cms.pod
--- openssl-1.1.1/doc.org/man1/cms.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/cms.pod	2018-09-11 22:08:34.790830156 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-cms,
-cms - CMS utility
+openssl-cms - CMS utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/crl2pkcs7.pod openssl-1.1.1/doc/man1/crl2pkcs7.pod
--- openssl-1.1.1/doc.org/man1/crl2pkcs7.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/crl2pkcs7.pod	2018-09-11 22:11:06.401959876 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-crl2pkcs7,
-crl2pkcs7 - Create a PKCS#7 structure from a CRL and certificates
+openssl-crl2pkcs7 - Create a PKCS#7 structure from a CRL and certificates
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/crl.pod openssl-1.1.1/doc/man1/crl.pod
--- openssl-1.1.1/doc.org/man1/crl.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/crl.pod	2018-09-11 22:10:43.631289563 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-crl,
-crl - CRL utility
+openssl-crl - CRL utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/dgst.pod openssl-1.1.1/doc/man1/dgst.pod
--- openssl-1.1.1/doc.org/man1/dgst.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/dgst.pod	2018-09-11 22:12:41.804768298 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-dgst,
-dgst - perform digest operations
+openssl-dgst - perform digest operations
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/dhparam.pod openssl-1.1.1/doc/man1/dhparam.pod
--- openssl-1.1.1/doc.org/man1/dhparam.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/dhparam.pod	2018-09-11 22:08:08.440054452 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-dhparam,
-dhparam - DH parameter manipulation and generation
+openssl-dhparam - DH parameter manipulation and generation
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/dsaparam.pod openssl-1.1.1/doc/man1/dsaparam.pod
--- openssl-1.1.1/doc.org/man1/dsaparam.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/dsaparam.pod	2018-09-11 22:09:26.799027812 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-dsaparam,
-dsaparam - DSA parameter manipulation and generation
+openssl-dsaparam - DSA parameter manipulation and generation
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/dsa.pod openssl-1.1.1/doc/man1/dsa.pod
--- openssl-1.1.1/doc.org/man1/dsa.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/dsa.pod	2018-09-11 22:09:37.069330144 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-dsa,
-dsa - DSA key processing
+openssl-dsa - DSA key processing
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/ecparam.pod openssl-1.1.1/doc/man1/ecparam.pod
--- openssl-1.1.1/doc.org/man1/ecparam.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/ecparam.pod	2018-09-11 22:09:50.416389716 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-ecparam,
-ecparam - EC parameter manipulation and generation
+openssl-ecparam - EC parameter manipulation and generation
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/ec.pod openssl-1.1.1/doc/man1/ec.pod
--- openssl-1.1.1/doc.org/man1/ec.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/ec.pod	2018-09-11 22:10:38.384468443 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-ec,
-ec - EC key processing
+openssl-ec - EC key processing
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/enc.pod openssl-1.1.1/doc/man1/enc.pod
--- openssl-1.1.1/doc.org/man1/enc.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/enc.pod	2018-09-11 22:07:38.422504142 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-enc,
-enc - symmetric cipher routines
+openssl-enc - symmetric cipher routines
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/engine.pod openssl-1.1.1/doc/man1/engine.pod
--- openssl-1.1.1/doc.org/man1/engine.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/engine.pod	2018-09-11 22:11:28.725950372 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-engine,
-engine - load and query engines
+openssl-engine - load and query engines
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/errstr.pod openssl-1.1.1/doc/man1/errstr.pod
--- openssl-1.1.1/doc.org/man1/errstr.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/errstr.pod	2018-09-11 22:07:46.686080735 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-errstr,
-errstr - lookup error codes
+openssl-errstr - lookup error codes
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/gendsa.pod openssl-1.1.1/doc/man1/gendsa.pod
--- openssl-1.1.1/doc.org/man1/gendsa.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/gendsa.pod	2018-09-11 22:12:17.117374895 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-gendsa,
-gendsa - generate a DSA private key from a set of parameters
+openssl-gendsa - generate a DSA private key from a set of parameters
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/genpkey.pod openssl-1.1.1/doc/man1/genpkey.pod
--- openssl-1.1.1/doc.org/man1/genpkey.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/genpkey.pod	2018-09-11 22:10:07.880237141 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-genpkey,
-genpkey - generate a private key
+openssl-genpkey - generate a private key
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/genrsa.pod openssl-1.1.1/doc/man1/genrsa.pod
--- openssl-1.1.1/doc.org/man1/genrsa.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/genrsa.pod	2018-09-11 22:11:11.535444327 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-genrsa,
-genrsa - generate an RSA private key
+openssl-genrsa - generate an RSA private key
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/list.pod openssl-1.1.1/doc/man1/list.pod
--- openssl-1.1.1/doc.org/man1/list.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/list.pod	2018-09-11 22:08:58.891539618 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-list,
-list - list algorithms and features
+openssl-list - list algorithms and features
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/nseq.pod openssl-1.1.1/doc/man1/nseq.pod
--- openssl-1.1.1/doc.org/man1/nseq.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/nseq.pod	2018-09-11 22:10:48.351428513 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-nseq,
-nseq - create or examine a Netscape certificate sequence
+openssl-nseq - create or examine a Netscape certificate sequence
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/ocsp.pod openssl-1.1.1/doc/man1/ocsp.pod
--- openssl-1.1.1/doc.org/man1/ocsp.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/ocsp.pod	2018-09-11 22:08:49.024582492 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-ocsp,
-ocsp - Online Certificate Status Protocol utility
+openssl-ocsp - Online Certificate Status Protocol utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/passwd.pod openssl-1.1.1/doc/man1/passwd.pod
--- openssl-1.1.1/doc.org/man1/passwd.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/passwd.pod	2018-09-11 22:12:23.680901443 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-passwd,
-passwd - compute password hashes
+openssl-passwd - compute password hashes
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/pkcs12.pod openssl-1.1.1/doc/man1/pkcs12.pod
--- openssl-1.1.1/doc.org/man1/pkcs12.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/pkcs12.pod	2018-09-11 22:12:08.627124963 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-pkcs12,
-pkcs12 - PKCS#12 file utility
+openssl-pkcs12 - PKCS#12 file utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/pkcs7.pod openssl-1.1.1/doc/man1/pkcs7.pod
--- openssl-1.1.1/doc.org/man1/pkcs7.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/pkcs7.pod	2018-09-11 22:08:44.624452964 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-pkcs7,
-pkcs7 - PKCS#7 utility
+openssl-pkcs7 - PKCS#7 utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/pkcs8.pod openssl-1.1.1/doc/man1/pkcs8.pod
--- openssl-1.1.1/doc.org/man1/pkcs8.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/pkcs8.pod	2018-09-11 22:10:34.057674407 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-pkcs8,
-pkcs8 - PKCS#8 format private key conversion tool
+openssl-pkcs8 - PKCS#8 format private key conversion tool
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/pkeyparam.pod openssl-1.1.1/doc/man1/pkeyparam.pod
--- openssl-1.1.1/doc.org/man1/pkeyparam.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/pkeyparam.pod	2018-09-11 22:09:32.382525509 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-pkeyparam,
-pkeyparam - public key algorithm parameter processing tool
+openssl-pkeyparam - public key algorithm parameter processing tool
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/pkey.pod openssl-1.1.1/doc/man1/pkey.pod
--- openssl-1.1.1/doc.org/man1/pkey.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/pkey.pod	2018-09-11 22:09:57.529932454 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-pkey,
-pkey - public or private key processing tool
+openssl-pkey - public or private key processing tool
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/pkeyutl.pod openssl-1.1.1/doc/man1/pkeyutl.pod
--- openssl-1.1.1/doc.org/man1/pkeyutl.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/pkeyutl.pod	2018-09-11 22:11:00.448451286 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-pkeyutl,
-pkeyutl - public key algorithm utility
+openssl-pkeyutl - public key algorithm utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/prime.pod openssl-1.1.1/doc/man1/prime.pod
--- openssl-1.1.1/doc.org/man1/prime.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/prime.pod	2018-09-11 22:10:55.561640764 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-prime,
-prime - compute prime numbers
+openssl-prime - compute prime numbers
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/rand.pod openssl-1.1.1/doc/man1/rand.pod
--- openssl-1.1.1/doc.org/man1/rand.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/rand.pod	2018-09-11 22:11:39.589603508 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-rand,
-rand - generate pseudo-random bytes
+openssl-rand - generate pseudo-random bytes
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/rehash.pod openssl-1.1.1/doc/man1/rehash.pod
--- openssl-1.1.1/doc.org/man1/rehash.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/rehash.pod	2018-09-11 22:08:28.943991372 +0200
@@ -5,8 +5,7 @@
 
 =head1 NAME
 
-openssl-c_rehash, openssl-rehash,
-c_rehash, rehash - Create symbolic links to files named by the hash values
+c_rehash, openssl-rehash - Create symbolic links to files named by the hash values
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/req.pod openssl-1.1.1/doc/man1/req.pod
--- openssl-1.1.1/doc.org/man1/req.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/req.pod	2018-09-11 22:08:54.174734102 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-req,
-req - PKCS#10 certificate request and certificate generating utility
+openssl-req - PKCS#10 certificate request and certificate generating utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/rsa.pod openssl-1.1.1/doc/man1/rsa.pod
--- openssl-1.1.1/doc.org/man1/rsa.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/rsa.pod	2018-09-11 22:12:34.427884475 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-rsa,
-rsa - RSA key processing tool
+openssl-rsa - RSA key processing tool
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/rsautl.pod openssl-1.1.1/doc/man1/rsautl.pod
--- openssl-1.1.1/doc.org/man1/rsautl.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/rsautl.pod	2018-09-11 22:11:19.782353762 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-rsautl,
-rsautl - RSA utility
+openssl-rsautl - RSA utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/s_client.pod openssl-1.1.1/doc/man1/s_client.pod
--- openssl-1.1.1/doc.org/man1/s_client.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/s_client.pod	2018-09-11 22:12:02.190268812 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-s_client,
-s_client - SSL/TLS client program
+openssl-s_client - SSL/TLS client program
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/sess_id.pod openssl-1.1.1/doc/man1/sess_id.pod
--- openssl-1.1.1/doc.org/man1/sess_id.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/sess_id.pod	2018-09-11 22:11:56.426765815 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-sess_id,
-sess_id - SSL/TLS session handling utility
+openssl-sess_id - SSL/TLS session handling utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/smime.pod openssl-1.1.1/doc/man1/smime.pod
--- openssl-1.1.1/doc.org/man1/smime.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/smime.pod	2018-09-11 22:10:19.807254909 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-smime,
-smime - S/MIME utility
+openssl-smime - S/MIME utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/speed.pod openssl-1.1.1/doc/man1/speed.pod
--- openssl-1.1.1/doc.org/man1/speed.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/speed.pod	2018-09-11 22:09:21.542206398 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-speed,
-speed - test library performance
+openssl-speed - test library performance
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/spkac.pod openssl-1.1.1/doc/man1/spkac.pod
--- openssl-1.1.1/doc.org/man1/spkac.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/spkac.pod	2018-09-11 22:10:29.570875660 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-spkac,
-spkac - SPKAC printing and generating utility
+openssl-spkac - SPKAC printing and generating utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/srp.pod openssl-1.1.1/doc/man1/srp.pod
--- openssl-1.1.1/doc.org/man1/srp.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/srp.pod	2018-09-11 22:10:02.696751220 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-srp,
-srp - maintain SRP password file
+openssl-srp - maintain SRP password file
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/s_server.pod openssl-1.1.1/doc/man1/s_server.pod
--- openssl-1.1.1/doc.org/man1/s_server.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/s_server.pod	2018-09-11 22:09:04.541705946 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-s_server,
-s_server - SSL/TLS server program
+openssl-s_server - SSL/TLS server program
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/s_time.pod openssl-1.1.1/doc/man1/s_time.pod
--- openssl-1.1.1/doc.org/man1/s_time.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/s_time.pod	2018-09-11 22:09:44.646219856 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-s_time,
-s_time - SSL/TLS performance timing program
+openssl-s_time - SSL/TLS performance timing program
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/storeutl.pod openssl-1.1.1/doc/man1/storeutl.pod
--- openssl-1.1.1/doc.org/man1/storeutl.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/storeutl.pod	2018-09-11 22:07:55.473006066 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-storeutl,
-storeutl - STORE utility
+openssl-storeutl - STORE utility
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/tsget.pod openssl-1.1.1/doc/man1/tsget.pod
--- openssl-1.1.1/doc.org/man1/tsget.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/tsget.pod	2018-09-11 22:10:15.233786945 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-tsget,
-tsget - Time Stamping HTTP/HTTPS client
+openssl-tsget - Time Stamping HTTP/HTTPS client
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/ts.pod openssl-1.1.1/doc/man1/ts.pod
--- openssl-1.1.1/doc.org/man1/ts.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/ts.pod	2018-09-11 22:10:24.110714927 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-ts,
-ts - Time Stamping Authority tool (client/server)
+openssl-ts - Time Stamping Authority tool (client/server)
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/verify.pod openssl-1.1.1/doc/man1/verify.pod
--- openssl-1.1.1/doc.org/man1/verify.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/verify.pod	2018-09-11 22:11:51.053274300 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-verify,
-verify - Utility to verify certificates
+openssl-verify - Utility to verify certificates
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/version.pod openssl-1.1.1/doc/man1/version.pod
--- openssl-1.1.1/doc.org/man1/version.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/version.pod	2018-09-11 22:11:45.803119749 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-version,
-version - print OpenSSL version information
+openssl-version - print OpenSSL version information
 
 =head1 SYNOPSIS
 
diff -urN openssl-1.1.1/doc.org/man1/x509.pod openssl-1.1.1/doc/man1/x509.pod
--- openssl-1.1.1/doc.org/man1/x509.pod	2018-09-11 14:48:22.000000000 +0200
+++ openssl-1.1.1/doc/man1/x509.pod	2018-09-11 22:08:15.893607199 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-x509,
-x509 - Certificate display and signing utility
+openssl-x509 - Certificate display and signing utility
 
 =head1 SYNOPSIS
 
--- openssl-1.1.1/doc/man5/config.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man5/config.pod	2018-09-13 18:55:53.379314240 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-config - OpenSSL CONF library configuration files
+openssl-config - OpenSSL CONF library configuration files
 
 =head1 DESCRIPTION
 
--- openssl-1.1.1/doc/man5/x509v3_config.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man5/x509v3_config.pod	2018-09-13 18:56:06.236359843 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-x509v3_config - X509 V3 certificate extension configuration format
+openssl-x509v3_config - X509 V3 certificate extension configuration format
 
 =head1 DESCRIPTION
 
--- openssl-1.1.1/doc/man7/bio.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/bio.pod	2018-09-13 19:13:57.454822064 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-bio - Basic I/O abstraction
+openssl-bio - Basic I/O abstraction
 
 =head1 SYNOPSIS
 
--- openssl-1.1.1/doc/man7/crypto.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/crypto.pod	2018-09-13 19:14:03.525002575 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-crypto - OpenSSL cryptographic library
+openssl-crypto - OpenSSL cryptographic library
 
 =head1 SYNOPSIS
 
--- openssl-1.1.1/doc/man7/ct.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/ct.pod	2018-09-13 19:14:07.405117959 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-ct - Certificate Transparency
+openssl-ct - Certificate Transparency
 
 =head1 SYNOPSIS
 
--- openssl-1.1.1/doc/man7/des_modes.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/des_modes.pod	2018-09-13 19:14:10.988557855 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-des_modes - the variants of DES and other crypto algorithms of OpenSSL
+openssl-des_modes - the variants of DES and other crypto algorithms of OpenSSL
 
 =head1 DESCRIPTION
 
--- openssl-1.1.1/doc/man7/evp.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/evp.pod	2018-09-13 19:14:15.148681567 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-evp - high-level cryptographic functions
+openssl-evp - high-level cryptographic functions
 
 =head1 SYNOPSIS
 
--- openssl-1.1.1/doc/man7/ssl.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/ssl.pod	2018-09-13 19:14:35.495953306 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-ssl - OpenSSL SSL/TLS library
+openssl-ssl - OpenSSL SSL/TLS library
 
 =head1 SYNOPSIS
 
--- openssl-1.1.1/doc/man7/x509.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/x509.pod	2018-09-13 19:14:40.996116867 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-x509 - X.509 certificate handling
+openssl-x509 - X.509 certificate handling
 
 =head1 SYNOPSIS
 
--- openssl-1.1.1/doc/man7/Ed25519.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/Ed25519.pod	2018-09-13 19:27:15.788862330 +0200
@@ -2,9 +2,7 @@
 
 =head1 NAME
 
-Ed25519,
-Ed448
-- EVP_PKEY Ed25519 and Ed448 support
+openssl-Ed25519, openssl-Ed448 - EVP_PKEY Ed25519 and Ed448 support
 
 =head1 DESCRIPTION
 
--- openssl-1.1.1/doc/man7/passphrase-encoding.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/passphrase-encoding.pod	2018-09-13 19:27:38.656220157 +0200
@@ -4,7 +4,7 @@
 
 =head1 NAME
 
-passphrase-encoding
+openssl-passphrase-encoding
 - How diverse parts of OpenSSL treat pass phrases character encoding
 
 =head1 DESCRIPTION
--- openssl-1.1.1/doc/man7/RAND_DRBG.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/RAND_DRBG.pod	2018-09-13 19:27:45.693099512 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-RAND_DRBG - the deterministic random bit generator
+openssl-RAND_DRBG - the deterministic random bit generator
 
 =head1 SYNOPSIS
 
--- openssl-1.1.1/doc/man7/RAND.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/RAND.pod	2018-09-13 19:27:53.023321066 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-RAND
-- the OpenSSL random generator
+openssl-RAND - the OpenSSL random generator
 
 =head1 DESCRIPTION
 
--- openssl-1.1.1/doc/man7/RSA-PSS.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/RSA-PSS.pod	2018-09-13 19:27:58.186810465 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-RSA-PSS - EVP_PKEY RSA-PSS algorithm support
+openssl - RSA-PSS - EVP_PKEY RSA-PSS algorithm support
 
 =head1 DESCRIPTION
 
--- openssl-1.1.1/doc/man7/scrypt.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/scrypt.pod	2018-09-13 19:28:04.880346108 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-scrypt - EVP_PKEY scrypt KDF support
+openssl-scrypt - EVP_PKEY scrypt KDF support
 
 =head1 DESCRIPTION
 
--- openssl-1.1.1/doc/man7/SM2.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/SM2.pod	2018-09-13 19:28:09.697158362 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-SM2 - Chinese SM2 signature and encryption algorithm support
+openssl-SM2 - Chinese SM2 signature and encryption algorithm support
 
 =head1 DESCRIPTION
 
--- openssl-1.1.1/doc/man7/X25519.pod~	2018-09-11 14:48:23.000000000 +0200
+++ openssl-1.1.1/doc/man7/X25519.pod	2018-09-13 19:28:23.594245064 +0200
@@ -2,9 +2,7 @@
 
 =head1 NAME
 
-X25519,
-X448
-- EVP_PKEY X25519 and X448 support
+openssl-X25519, openssl-X448 - EVP_PKEY X25519 and X448 support
 
 =head1 DESCRIPTION
 
