--- openssl-0.9.7m/crypto/cryptlib.h~	2005-04-13 11:49:51.000000000 +0300
+++ openssl-0.9.7m/crypto/cryptlib.h	2011-11-06 01:07:47.482927980 +0200
@@ -77,7 +77,7 @@
 #ifndef OPENSSL_SYS_VMS
 #define X509_CERT_AREA		OPENSSLDIR
 #define X509_CERT_DIR		OPENSSLDIR "/certs"
-#define X509_CERT_FILE		OPENSSLDIR "/cert.pem"
+#define X509_CERT_FILE		"/etc/certs/ca-certificates.crt"
 #define X509_PRIVATE_DIR	OPENSSLDIR "/private"
 #else
 #define X509_CERT_AREA		"SSLROOT:[000000]"
