--- openssl-1.1.0g/crypto/include/internal/cryptlib.h.orig	2017-11-02 15:29:03.000000000 +0100
+++ openssl-1.1.0g/crypto/include/internal/cryptlib.h	2017-11-18 21:05:07.408322386 +0100
@@ -42,7 +42,7 @@
 # ifndef OPENSSL_SYS_VMS
 #  define X509_CERT_AREA          OPENSSLDIR
 #  define X509_CERT_DIR           OPENSSLDIR "/certs"
-#  define X509_CERT_FILE          OPENSSLDIR "/cert.pem"
+#  define X509_CERT_FILE          "/etc/certs/ca-certificates.crt"
 #  define X509_PRIVATE_DIR        OPENSSLDIR "/private"
 #  define CTLOG_FILE              OPENSSLDIR "/ct_log_list.cnf"
 # else
